### PR TITLE
feature/AT-RSC-UI-Members

### PR DESCRIPTION
**Pull request by hgglez**

### CODE REVIEW

**Comment by hgglez** [comment]
update files

File: src/routes/at-resources.routes.js [hunk 1]
```diff
@@ -38,6 +38,20 @@ router.get("/task/edit/:id", renderEditTaskForm);
...
+Router;
+
+//Controller
+
+const { renderMembersList } = require
+('../controllers/members.controller')
+
+// Gets
+
+router.get("/members", renderMembersList);
```
Comment (at-internship) [inline src/routes/at-resources.routes.js]: use `router.get("/members", members);`

File: src/server.js [hunk 2]
```diff
@@ -29,33 +29,36 @@ app.use(morgan("dev"));
...
+  next();
 });
 
 // Global variables
 
-// AT-RESOURCES Routes - General 
-app.use("/", require("./routes/at-resources.routes"));
+// Routes
+
+app.use("/members", require("./routes/members.routes");)
```
Comment (at-internship) [inline src/server.js]: You don't need this route

File: src/views/memberslist.hbs [hunk 3]
```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet consectetur, adipisicing elit. Maxime expedita sequi voluptas quisquam, et odit. Explicabo nisi recusandae dolores alias doloremque deleniti vel molestias laboriosam. Sequi, ipsam obcaecati! Nisi, quasi!
```
Comment (at-internship) [inline src/views/memberslist.hbs]:
```
<div class="container-fluid">
    <div class="row">
        <nav id="sidebarMenu" class="col-md-3 col-lg-2 d-md-block bg-light sidebar collapse">
            <div class="sidebar-sticky pt-3">
                <!-- MENU-AT-RESOURCES -->
                {{> menu-at-resources }}
                <!-- MENU-AT-RESOURCES -->
            </div>
        </nav>
            <!-- MEMBERS -->
        <main role="main" class="col-md-9 ml-sm-auto col-lg-10 px-md-5">
            <div
                class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pt-3 pb-2 mb-3 border-bottom">
                <h1 class="h2">Members</h1>  
            </div>
        </main>
        <!-- MEMBERS -->
    </div>
</div>
```

File: src/routes/at-resources.routes.js [hunk 4]
```diff
@@ -38,6 +38,20 @@ router.get("/task/edit/:id", renderEditTaskForm);
 // AT-RESOURCES - updateTask
 router.put("/task/edit/:id", updateTask);
 
+const router = require("express"),
+Router;
+
+//Controller
+
+const { renderMembersList } = require
+('../controllers/members.controller')
```
Comment (at-internship) [inline src/routes/at-resources.routes.js]: you don't need `const { renderMembersList } = require('../controllers/members.controller')`

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/routes/at-resources.routes.js (hunk 1 above)
Comment (hgglez) [reply src/routes/at-resources.routes.js]: Done

File: src/server.js (hunk 2 above)
Comment (hgglez) [reply src/server.js]: Done

File: src/views/memberslist.hbs (hunk 3 above)
Comment (hgglez) [reply src/views/memberslist.hbs]: Done

File: src/routes/at-resources.routes.js (hunk 4 above)
Comment (hgglez) [reply src/routes/at-resources.routes.js]: Done

**Review by raaul-at — APPROVED** [review]

**Review by irvingesp2190 — APPROVED** [review]

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**